src/xunit.performance.api/PerformanceMonitorCounter/BasePerformanceMonitorCounter.cs

```diff
@@ -17,7 +17,7 @@ public override IEnumerable<ProviderInfo> ProviderInfo
             get
             {
                 yield return new KernelProviderInfo() {
-                    Keywords = unchecked((ulong)(KernelTraceEventParser.Keywords.PMCProfile | KernelTraceEventParser.Keywords.Profile)),
+                    Keywords = unchecked((ulong)KernelTraceEventParser.Keywords.PMCProfile),
```

Comment:
I assume that PMCProfile was already setting all of the bits that Profile was? Or is this a functional change?

Reply:
`PMCProfile` is not a default anymore, but only set when Instructions Retired, Cache Misses or Branch Mispredictions are specified.

Reply:
`Profile` is a default flag (always captured). I removed some of the redundancy here, and flags that were set but not used.

Reply:
Yes, I understand.